pom.xml

```diff
@@ -126,6 +127,96 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
```

Comment:
I don't know what the point of this is but I don't like it. dependencies of sub projects do not belong in parent pom. the parent poms primary purpose is to share common configs. acf-bukkit has no need for information on acf-sponge.

Reply:
This is exactly sharing common configs. What this section does is declaring common dependency properties in patent so that those should not be explicitly specified (unless those values should be overridden) in children. `dependencyManagement` by itself will not add those dependencys to project (for this you've got the very `dependencies` tag in patent) but will simply declare that if the child does not specify (e.g. and most commonly) the version of dependency `bar.group-id:foo-artifact-id` then it should be looked up in parent's dependency management section.

The reason why I've put ACF-dependencies in it too is simply to let them also be used (at current it seems to happen inky for ACF-core and ACF-bukkit) in dependent modules without  a need to declare their versions.

Reply:
As of using variables in `<version>` of module, it is a deprecated feature (Maven will allow it but will complain in logs) which is not even needed as children inherit parent's version.